notagain/lua/notagain/pac_server/autorun/discordrelay.lua

```diff
@@ -548,9 +561,9 @@ else
 		end
 
 		if nick ~= "" then
-			chat.AddText(Color(222,129,199),"[Discord] "..nick, Color(255,255,255,255),": ",message)
+			chat.AddText( Color(114,137,218,255), "[Discord] "..nick, Color(255,255,255,255),": "..message)
```

Comment:
Good choice. Make it hard to difference from [chat and discord](http://i.imgur.com/KsoubIr.png)
Some people would struggle with that so you might wanna make it more outstanding